FAQ.md

```diff
@@ -242,6 +242,16 @@ generate WebAssembly code from C and C++, and will use LLVM IR in their
 implementation similarly to how PNaCl and Emscripten do today.
 
 
+## Why not use the .net or jvm virtual machines and bytecodes?
+
+Simply, web browsers already have their own virtual machines for executing 
+platform-independent code, and integrating the existing infrastructure of .net 
+or the jvm into browser environments on top of these existing systems would 
+incur massive costs for end-users. Also, the design goals in jvm and .net's 
```

Comment:
"costs" could be misinterpreted as a monetary expense, which although being a legitimate fear given the history of both Java and .NET, would be unlikely.  Regardless, I'd like to see a less ambiguous word or phrase here, but not sure what it would be...

Also, JVM and .NET should be all caps.